pkg/k8shandler/status.go

```diff
@@ -49,3 +54,35 @@ func updateNodeStatus(node *nodeState, dpl *v1alpha1.ElasticsearchStatus) {
 	}
 	dpl.Nodes = append(dpl.Nodes, nodeStatus)
 }
+
+func clusterHealth() string {
+	config := &ExecConfig{
+		namespace:      "openshift-logging",
+		labelName:      "component",
+		labelPrefix:    "elasticsearch",
+		containerName:  "elasticsearch",
+		command:        []string{"es_util", "--query=_cluster/health?pretty=true"},
+		kubeConfigPath: "/home/origin/admin.kubeconfig",
+		masterURL:      "https://ip-172-18-8-136.ec2.internal:8443",
```

Comment:
Shouldn't this be the 'well known' master url  or discoverable by env vars that are added to all pods?

Reply:
@josefkarasek these vars are always there or we could use those described in the kube doc.

Reply:
https://github.com/operator-framework/operator-sdk/blob/e5a0ab096e1a7c0e6b937d2b41707eccb82c3c77/pkg/test/framework.go#L51
https://github.com/kubernetes/client-go/blob/1f13a808da65775f22cbf47862c4e5898d8f4ca1/rest/config.go#L50-L54